src/main/scala/com/github/unchama/seichiassist/listener/PlayerInventoryListener.scala

```diff
@@ -572,5 +574,57 @@ class PlayerInventoryListener(implicit effectEnvironment: EffectEnvironment,
     } else if (topinventory.getTitle == LIGHT_PURPLE.toString + "" + BOLD + "スキルを進化させました") {
       event.setCancelled(true)
     }
+
   }
+
+  /**
+   * 名義除去システム
+   */
+  @EventHandler
+  def onItemOwnerNameRemoveEvent(event: InventoryCloseEvent): Unit = {
+    val player = event.getPlayer match {
+      case p: Player => p
+      case _ => return
+    }
+    val inventory = event.getInventory
+
+    //インベントリサイズが36、あるいはインベントリのタイトルが予期したものでなければ処理を終了させる
+    if (inventory.row != 4 || inventory.getTitle != s"$GOLD${BOLD}所有者名義を削除したいアイテムを投入してください") return
+    val items = inventory.getContents
+
+    var count = 0
+    //for文を使い、1つずつアイテムを見ていく
+    for (item <- items) {
+      if (item != null) {
+        if (item.hasItemMeta && item.getItemMeta.hasLore) {
+          if (Util.itemStackContainsOwnerName(item, player.getName)) {
+            val itemLore = item.getItemMeta.getLore.asScala.toList
+            //itemLoreのListの中から、"所有者"で始まるものを弾き、新しく「所有者:なし」を付け加えたLoreをアイテムにつける
+            val newItemLore = itemLore.map(lore =>
+              if (lore.contains("所有者")) "所有者:なし"
+              else lore
+            ).asJava
+            val itemMeta = Bukkit.getItemFactory.getItemMeta(item.getType).tap { meta =>
+              import meta._
+              //一度取り消してから新しくLoreを付与しないとついていたLoreが削除されず、Loreが二重になる?
+              getLore.clear()
+              setLore(newItemLore)
+            }
+            val itemStack = new ItemStack(item.getType, item.getAmount)
+            itemStack.setItemMeta(itemMeta)
+            count += 1
+          }
+        }
+        Util.grantItemStacksEffect[IO](item)
+      }
+    }
+    if (count < 1) {
+      player.sendMessage(s"{GREEN}所有者表記のされたアイテムが認識されませんでした。すべてのアイテムを返却します。")
+    } else {
+      player.sendMessage(s"{GREEN} ${count}個のアイテムのを認識し、所有者表記を「なし」に変更しました")
```

Comment:
```suggestion
      player.sendMessage(s"{GREEN} ${count}個のアイテムを認識し、所有者表記を「なし」に変更しました")
```

please fix this potential typo

Reply:
見落としてました...ありがとうございます 901cbf3 にて修正しました